src/message.ts

```diff
@@ -414,6 +414,59 @@ export class Message implements Sayable {
     return fromId === userId
   }
 
+  /**
+   *
+   * Get the mentioned contact which the message is mentioned for.
+   * @returns {Contact[]} return the contactList which the message is mentioned for
+   *
+   * @example
+   * ```ts
+   * const content = message.content()
+   * const contactList = message.mention()
+   * console.log(content)
+   * console.log(contactList)
+   * ```
+   */
+  public mention(): Contact[] {
```

Comment:
This may be an English issue, First I see the method name, I thought it would be `mention(somebody, something)`. but after I read the comments, I realize it's getting a mentioned list. 
I know `mention` can be a noun, but still make me confused. If it doesn't make you guys confused, then ignore this comment.